liquidator/index.js

```diff
@@ -24,9 +24,10 @@ const ExpandedERC20 = artifacts.require("ExpandedERC20");
  *     mode which will exit after the loop.
  * @param {Object} priceFeedConfig Configuration to construct the price feed object.
  * @param {Object} [liquidatorConfig] Configuration to construct the liquidator.
+ * @param {String} [liquidatorOverridePrice] String encoded BigNumber to override the liquidator price feed.
  * @return None or throws an Error.
  */
-async function run(address, pollingDelay, priceFeedConfig, liquidatorConfig) {
+async function run(address, pollingDelay, priceFeedConfig, liquidatorConfig, liquidatorOverridePrice = null) {
```

Comment:
Just wanted to flag this `= null` to add to @nicholaspai's comment above. To be clear, I'm not super opinionated. I think either way we go, we should be consistent. The `= null` is a bit more explicit so it could improve readability.

Reply:
if we do that then we should apply the same rule to all optional parameters. There are not that many but it would make it explicitly clear what is optional.

We currently do use defaults in a few places like polling delays and in the gasestimator among others.

Reply:
For now, I removed the `= null` and we can choose to add be more aggressive defaulting in the future. Sounds good @nicholaspai ?